resources/s3_bucket.rb

```diff
@@ -19,7 +19,7 @@
     end
   else # create the bucket from scratch
     converge_by "create S3 bucket #{new_resource.name}" do
-      s3_bucket.create(new_resource.name)
+      s3_bucket.create
```

Comment:
This seems to be an unrelated change, though I think it’s beneficial, since the name of the bucket is set on [line 83](https://github.com/chef-cookbooks/aws/pull/342/files#diff-1454e2db6eef6f80f89776f6ce13ad47R83).

Reply:
IIRC, it works the same with or without the parameter in v2, but doesn't work with the parameter in v3